crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -166,11 +162,17 @@ pub enum IatapayPaymentStatus {
 impl From<IatapayPaymentStatus> for enums::AttemptStatus {
     fn from(item: IatapayPaymentStatus) -> Self {
         match item {
-            IatapayPaymentStatus::Authorized | IatapayPaymentStatus::Settled => Self::Charged,
-            IatapayPaymentStatus::Failed | IatapayPaymentStatus::UnexpectedSettled => Self::Failure,
+            IatapayPaymentStatus::Authorized => Self::Authorized,
+            IatapayPaymentStatus::Settled
+            | IatapayPaymentStatus::Cleared
+            | IatapayPaymentStatus::Tobeinvestigated
+            | IatapayPaymentStatus::Blocked => Self::Charged,
+            IatapayPaymentStatus::Failed
+            | IatapayPaymentStatus::UnexpectedSettled
+            | IatapayPaymentStatus::Unknown => Self::Failure,
             IatapayPaymentStatus::Created => Self::AuthenticationPending,
-            IatapayPaymentStatus::Initiated => Self::Pending,
-            _ => Self::Voided,
+            IatapayPaymentStatus::Initiated | IatapayPaymentStatus::Locked => Self::Pending,
```

Comment:
As per the screenshot attached in PR description Locked is applicable only for refunds